modules/passkey/package.json

```diff
@@ -37,34 +36,33 @@
     "deploy": "hardhat deploy --network",
     "fmt": "prettier --write .",
     "fmt:check": "prettier --check .",
-    "lint": "npm run lint:sol && npm run lint:ts",
+    "lint": "pnpm run lint:sol && pnpm run lint:ts",
     "lint:sol": "solhint 'contracts/**/*.sol'",
     "lint:ts": "eslint .",
     "test": "hardhat test",
     "test:4337": "4337-local-bundler-test",
     "test:4337:upstream": "USE_UPSTREAM_BUNDLER=1 4337-local-bundler-test",
-    "test:all": "npm run test && npm run test:4337",
-    "prepare": "npm run build -w ../4337 && npm run build"
+    "test:all": "pnpm run test && pnpm run test:4337",
+    "prepare": "pnpm run build"
   },
   "devDependencies": {
-    "@account-abstraction/contracts": "^0.7.0",
     "@noble/curves": "^1.4.0",
     "@nomicfoundation/hardhat-toolbox": "^5.0.0",
+    "@nomicfoundation/hardhat-ethers": "^3.0.6",
+    "@nomicfoundation/hardhat-network-helpers": "^1.0.10",
     "@safe-global/mock-contract": "^4.1.0",
-    "@safe-global/safe-4337": "^0.3.0",
-    "@safe-global/safe-4337-local-bundler": "^0.0.0",
+    "@safe-global/safe-4337": "workspace:^0.3.0",
+    "@safe-global/safe-4337-local-bundler": "workspace:^0.0.0",
     "@simplewebauthn/server": "^10.0.0",
     "dotenv": "^16.4.5",
+    "ethers": "^6.12.1",
     "hardhat": "^2.22.3",
     "hardhat-deploy": "^0.12.4",
-    "solhint": "^4.5.4"
-  },
-  "overrides": {
-    "@safe-global/safe-contracts": {
-      "ethers": "^6.11.1"
-    }
+    "solhint": "^5.0.1"
   },
   "dependencies": {
+    "@openzeppelin/contracts": "^5.0.0",
+    "@account-abstraction/contracts": "^0.7.0",
     "@safe-global/safe-contracts": "^1.4.1-build.0",
```

Comment:
Hmm, why is this not a `devDependency` as well?

Reply:
It is used in the launchpad contracts and required if someone will use the package and build the contracts from source.